Tests/OptimizelyTests-Common/DatafileHandlerTests.swift

```diff
@@ -303,6 +303,22 @@ class DatafileHandlerTests: XCTestCase {
         // finally remove the datafile when complete.
         try? FileManager.default.removeItem(at: fileUrl!)
     }
+    
+    func testSetPeriodicInterval() {
+        let handler = DefaultDatafileHandler()
+        XCTAssert(handler.timers.property!.keys.isEmpty)
+        
+        handler.setPeriodicInterval(sdkKey: "abc", interval: 123)
```

Comment:
it would be great if we could test it for multiple sdkKeys.

Reply:
@yasirfolio3 Agreed. We'll add them when it's fixed to fully support multiple sdkKeys.